packages/docs-app/src/components/navbarActions.tsx

```diff
@@ -79,17 +85,17 @@ export class NavbarActions extends React.PureComponent<INavbarActionsProps, {}>
         const libs = releases.filter(({ name }) => COMPONENT_PACKAGES.indexOf(name) >= 0).map(renderItem);
         const tooling = releases.filter(({ name }) => COMPONENT_PACKAGES.indexOf(name) === -1).map(renderItem);
         return (
-            <Menu className="docs-releases-menu">
+            <Menu>
                 <MenuItem
                     href="https://github.com/palantir/blueprint/releases"
                     iconName="book"
                     target="_blank"
                     text="Release notes"
                 />
-                <MenuDivider title="Components" />
+                <MenuDivider title="Component packages" />
                 {libs}
-                <MenuDivider title="Tooling" />
-                {tooling}
+                <MenuDivider />
+                <MenuItem text="Infrastructure packages">{tooling}</MenuItem>
```

Comment:
I like "Build tooling" as mentioned in the README.

All of Blueprint is considered "product infrastructure".

Reply:
sure "Build tooling" is nice